### PR TITLE
Update spike-protection.mdx

### PR DESCRIPTION
**Pull request by lizokm**
Add an explanation about the minimum duration of spikes that's required in order for spikes to be visible in the stats page.

### CODE REVIEW

File: src/docs/product/accounts/quotas/spike-protection.mdx [hunk 1]
```diff
@@ -60,7 +60,11 @@ We recommend taking the following steps to manage your spikes:
...
 - Set up [spend allocations](/product/accounts/quotas/#spend-allocation) to ensure your critical projects are guaranteed a portion of your reserved volume, even if there are spikes in other projects.
 
-To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the desired event in the "Category" dropdown. You can see project-level details of your stats by using the project selector. Sentry can show the spike threshold for a time duration greater than 6hrs and less than 30 days.
+To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the event of interest from the "Category" dropdown. You'll be able to see project-level details of your stats by using the project selector.
+
+Note that only spikes that are over an hour long at minimum will be viewable from the stats page. A spike threshold in the form of a dotted line will be viewable for spikes that last more than 6hrs and less than 30 days.
```
Comment (lizokm) [inline src/docs/product/accounts/quotas/spike-protection.mdx:65]:
```suggestion
To ensure accuracy, only spikes that are over an hour long and fall into a time range that's between 6 hours and 30 days will be viewable from the **Stats** page. You'll be able to gage any spikes against a spike threshold in the form of a dotted line as long as the time range is between 6 hours and 30 days.
```

File: src/docs/product/accounts/quotas/spike-protection.mdx [hunk 2]
```diff
@@ -60,7 +60,11 @@ We recommend taking the following steps to manage your spikes:
...
 
-To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the desired event in the "Category" dropdown. You can see project-level details of your stats by using the project selector. Sentry can show the spike threshold for a time duration greater than 6hrs and less than 30 days.
+To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the event of interest from the "Category" dropdown. You'll be able to see project-level details of your stats by using the project selector.
+
+To ensure accuracy, only spikes that are over an hour long and fall into a time range that's between 6 hours and 30 days will be viewable from the **Stats** page. You'll be able to gage any spikes against a spike threshold in the form of a dotted line as long as the time range is between 6 hours and 30 days.
```
Comment (cathteng) [inline src/docs/product/accounts/quotas/spike-protection.mdx:65]: can we mention the time period on the stats page specifically? the time range mentioned here almost sounds like the spike length or the spike threshold length

do we want to mention ongoing spikes?

**Comment by Dhrumil-Sentry** [comment]
@cathteng I thought we'll be making changes to the doc that explains Spike Protection algorithm to mention how the rate limit is applied?

**Comment by cathteng** [comment]
@Dhrumil-Sentry this is for the change you asked for a bit ago for clarifying how spikes show up in the stats page vs the audit log

File: src/docs/product/accounts/quotas/spike-protection.mdx (hunk 2 above)
Comment (lizokm) [inline src/docs/product/accounts/quotas/spike-protection.mdx:65]:
```suggestion

To ensure accuracy, your time window on the **Stats** page must be between a minimum of 6 hours to a maximum of 30 days for spikes and a spike threshold to appear. In addition, only spikes that are over an hour long will be viewable. 

```

**Review by cathteng — APPROVED** [review]

File: src/docs/product/accounts/quotas/spike-protection.mdx [hunk 3]
```diff
@@ -60,7 +60,11 @@ We recommend taking the following steps to manage your spikes:
...
 - Set up [spend allocations](/product/accounts/quotas/#spend-allocation) to ensure your critical projects are guaranteed a portion of your reserved volume, even if there are spikes in other projects.
 
-To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the desired event in the "Category" dropdown. You can see project-level details of your stats by using the project selector. Sentry can show the spike threshold for a time duration greater than 6hrs and less than 30 days.
+To review the events that were dropped to save your quota with Spike Protection, go to the "Usage Stats" tab of the **Stats** page for your Sentry org and select the event of interest from the "Category" dropdown. You'll be able to see project-level details of your stats by using the project selector.
+
+To ensure accuracy, your time window on the **Stats** page must be between a minimum of 6 hours to a maximum of 30 days for spikes and a spike threshold to appear. In addition, only spikes that are over an hour long will be viewable.
```
Comment (lizokm) [inline src/docs/product/accounts/quotas/spike-protection.mdx:65]:
```suggestion
To ensure accuracy, your time window on the **Stats** page must be between a minimum of 6 hours, to a maximum of 30 days for spikes and a spike threshold to appear. In addition, only spikes that are over an hour long will be viewable.
```

**Pull request merged**